.travis.yml

```diff
@@ -178,7 +178,7 @@ matrix:
 #    - UNSET_S3_SECRETS=1
 
 before_install:
-  - NOSE_OPTS=( )
+  - NOSE_OPTS=( -v )
```

Comment:
`git blame` or log might tell why we removed `-v` generally (may be was getting too long?)

Reply:
Needed it to debug the remote tests. It is removed now.